src/components/widgets/PAutoComplete.js

```diff
@@ -1,26 +1,39 @@
 import { PInput, PListItem, PList } from '../elements'
 import React from 'react'
+
+const capitalizeFirstLetter = string => string.charAt(0).toUpperCase() + string.slice(1)
+
 export default (props) => {
   return (
-    <div>
+    <div
+      className="auto-complete"
+    >
       <PInput
+        disabled={props.loading}
```

Comment:
Disable input while loading can prevent unexpected load but it hurts the user experience.

Reply:
I already applied another technique which is silent totally and won't hurt the UX anymore. Hope you like it.